spec/sidekiq/decision_review/sc_status_updater_job_spec.rb

```diff
@@ -532,7 +25,7 @@
       end
 
       it 'does not query SavedClaim::HigherLevelReview records' do
-        expect(SavedClaim::HigherLevelReview).not_to receive(:where)
+        expect(SavedClaim::SupplementalClaim).not_to receive(:where)
```

Comment:
The test name doesn't match the assertion?

Reply:
Bad copypasta test name, thanks